index.html

```diff
@@ -0,0 +1,144 @@
+            <h2>Main Program</h2>
+            <hr>
+            <div class="progg">
+                <div class="prog">
+                    <img src="./img/prog1.png" alt="program icon">
+                    <h3>Lecture </h3>
+                    <p> Meet the Keynote Speakers as they take turns to speak on pushing for a better world where we can
+                        continue to eat, play and connect. </p>
+                </div>
+                <div class="prog">
+                    <img src="./img/prog2.png" alt="program icon">
+                    <h3>Exhibitions</h3>
+                    <p> Inspections of an array of Bole and perculiar to various regions where chefs would give
+                        illustrate
+                        talks on how best their bole is served </p>
+                </div>
+                <div class="prog">
+                    <img src="./img/prog3.png" alt="program icon">
+                    <h3>Forum </h3>
+                    <p> chefs, stakeholders and participants interactive sessions, opinions and ideas would b e
+                        exchanged.
+                    </p>
+                </div>
+                <div class="prog">
+                    <img src="./img/prog4.png" alt="program icon">
+                    <h3>merchandise </h3>
+                    <p> Get the latest street Tech innovations.We have put together a Limited merch collections.</p>
+                </div>
+                <div class="prog">
+                    <img src="./img/prog5.png" alt="program icon">
+                    <h3>workshop</h3>
+                    <p> Introduction to new technological advancements in the processing of our local Bole</p>
+                </div>
+            </div>
+            <div class="join-bole">
+                <h4> Join Bole Festival 2033</h4>
+            </div>
+        </section>
+        <section class="featured-chefs">
+            <h2> Bole Festival Chefs</h2>
+            <hr>
+            <div class="chefz">
+            </div>
+            <button class="morebtn">
+                <h4>More <span><i class="ri-arrow-drop-down-line"></i></span></h4>
+            </button>
+        </section>
+    </main>
+    <footer>
+        <section class="partners">
+            <h4>
+                Sponsored By
+            </h4>
+            <hr>
+
+            <div class="partners-logo" id="sponsors">
+                <img src="./img/GTBank_logo.png" alt="gtbank">
+                <img src="./img/coker.png" alt="cocacola">
+                <img src="./img/Government_of_Rivers_State_logo.png" alt="Rs-govt">
+                <img src="./img/Rhythm_937_FM.png" alt="93.7fm">
+                <img src="./img/McDonald's_logo.png" alt="McDonald">
+            </div>
+        </section>
+
+        <section class="company">
+            <img src="./img/bolefest.png" alt="bole festival">
+            <p>2023 Bole Festival Port Harcourt <br>some Rights Reserved.</p>
+        </section>
```

Comment:
- [ ]  As it is suggested the logo should be aligned left and the text to the right side of the page. but to me it seemed that the texted is at the center of the page. please move the text to he right side to match the design. 
![image](https://user-images.githubusercontent.com/43877689/227660646-c0231de7-5e4d-4101-abd7-1be18975d459.png)